src/app/components/ModeRules.js

```diff
@@ -1,9 +1,9 @@
 const rulesTexts = {
-  people:
+  People:
     'You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select who from Star Wars is showed on the left (Jar Jar Binks right now) from available options.',
-  vehicles:
+  Vehicles:
     'You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select which vehicle from Star Wars is showed on the left.',
-  starships:
+  Starships:
     'You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select which starship from Star Wars is showed on the left.',
 };
 
```

Comment:
why?

Reply:
if keys collide with eg variables names we can use destructuring like this
`const {people: newPeopleNameInCurrentScope} = rulesTexts;`

Reply:
Solved with using constants